infra/notifications/src/app.js

```diff
@@ -471,6 +472,35 @@ app.post('/send_pn', pushAppAuth, async (req, res) => {
   res.send(data)
 })
 
+const cachedCountryCodes = []
+const cachcedLangCodes = []
+app.get('/push-targets', pushAppAuth, async (req, res) => {
+  if (!cachcedLangCodes.length) {
+    const [langCodes] = await db.sequelize.query(`
+      select distinct data->>'language' as langcode 
+        from growth_participant order by data->>'language' ASC;
+    `)
+
+    cachcedLangCodes.push(...langCodes.map(l => l.langcode).filter(l => l))
+  }
+
+  if (!cachedCountryCodes.length) {
+    const [countryCodes] = await db.sequelize.query(`
```

Comment:
Pretty slow query. If performance becomes an issue, consider adding a DB index on the country column.